lib/index.js

```diff
@@ -50,14 +52,6 @@ Clocker.getDataObject = function (row) {
   return obj
 }
 
-Clocker.formatElapsed = function (elapsed) {
```

Comment:
The function `formatElapsed` is currently only used for printing purposes. Therefore I thought it might be a good idea to make it a static property of `Clocker`, so it could be easily adjusted by the user if needed. This does not prevent you from putting it into `lib/util.js`, but I would suggest to use it as `Clocker.formatElapsed = formatElapsed` and later refer to it as `Clocker.formatElapsed` instead of `formatElapsed` again.